src/lightfast_mcp/servers/figma/websocket_server.py

```
@@ -0,0 +1,540 @@
+        client.metadata["last_document_info"] = document_info
+        client.metadata["last_document_update"] = time.time()
+
+        # No response needed for updates
+        return None
+
+    async def _handle_document_info_response(
+        self, client: FigmaClient, data: Dict[str, Any]
+    ) -> Optional[Dict[str, Any]]:
+        """Handle document info response from Figma plugin."""
+        document_info = data.get("document_info", {})
+        request_id = data.get("request_id")
```

Ruff (0.11.9)

Local variable `request_id` is assigned to but never used

Remove assignment to unused variable `request_id`

(F841)